Sources/Charcoal/Filters/Root/Views/CalloutOverlay.swift

```diff
@@ -120,3 +125,9 @@ final class CalloutOverlay: UIView {
         delegate?.calloutOverlayDidTapInside(self)
     }
 }
+
+private extension UIColor {
+    class var overlayBackground: UIColor {
+        dynamicColorIfAvailable(defaultColor: .milk, darkModeColor: .darkIce)
```

Comment:
I guess you can define it like

```
dynamicColorIfAvailable(defaultColor: .milk.withAlphaComponent(0.8), darkModeColor: .darkIce.withAlphaComponent(0.6))
```

Reply:
Aha, it actually works when I remove the `.` . Didn't know. 😛 Updated now.